### PR TITLE
Usage example/draw line on bitmap

### PR DESCRIPTION
**Pull request by NoahJCross**
This usage example creates a window and draws a green line on a bitmap from a start point to an end point, with red circles marking both points.

Splashkit Function: draw_line_on_bitmap
- [x] Title and explanation (.txt)
- [x] C++ code
- [x] C# code (top-level statements)
- [x] C# code (Object-Oriented Programming)
- [x] Python code
- [x] Screenshot
- [x] Simple, clear demonstration of the function
- [x] Code uses Splashkit functions
- [x] Tested in Chrome and Firefox

### CODE REVIEW

**Comment by breezy-codes** [comment]
Reviewed prior PR - https://github.com/thoth-tech/splashkit.io-starlight/pull/219

File: public/usage-examples/graphics/draw_line_on_bitmap/draw_line_on_bitmap-1-map-route-oop.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            SplashKit.DrawLineOnBitmap(bitmap, Color.Green,
+                          100, 80,    // Starting point (x1, y1)
+                          300, 220);  // End point (x2, y2)
+            SplashKit.FillCircleOnBitmap(bitmap, Color.Red, 100, 80, 5);    // Start point
+            SplashKit.FillCircleOnBitmap(bitmap, Color.Red, 300, 220, 5);   // End point
+
+            while (!window.CloseRequested)
+            {
+                SplashKit.ProcessEvents();
+                // Draw the bitmap to the window
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_line_on_bitmap/draw_line_on_bitmap-1-map-route-oop.cs:26]: This code comment is to different than what is in the other programs. Please update this.

File: public/usage-examples/graphics/draw_line_on_bitmap/draw_line_on_bitmap-1-map-route-oop.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        public static void Main()
+        {
+            // Create a Window and bitmap for the map
+            Window window = new Window("Window", 400, 300);
+            Bitmap bitmap = new Bitmap("map", 400, 300);
+
+            // Fill background with white for map background
+            SplashKit.ClearBitmap(bitmap, Color.White);
+
+            // Draw the route line and start and end points
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_line_on_bitmap/draw_line_on_bitmap-1-map-route-oop.cs:16]: Same with this line

File: public/usage-examples/graphics/draw_line_on_bitmap/draw_line_on_bitmap-1-map-route-oop.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create a Window and bitmap for the map
+            Window window = new Window("Window", 400, 300);
+            Bitmap bitmap = new Bitmap("map", 400, 300);
+
+            // Fill background with white for map background
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_line_on_bitmap/draw_line_on_bitmap-1-map-route-oop.cs:13]: So is this comment

**Review by breezy-codes — CHANGES_REQUESTED** [review]
Please update the code comments in the OOP file to match how the other ones are.

**Review by breezy-codes — APPROVED** [review]
I've reviewed this pull request and ran all the codes in my own environment, they all ran correctly free from any errors. The code comments are consistent over all files and structure remains the same.


- [X] All required files are present.
  - [X] Title and explanation (.txt)
  - [X] C++ code
  - [X] C# code (top-level statements)
  - [X] C# code (Object-Oriented Programming)
  - [X] Python code
- [X] Code correctly uses Splashkit functions.
- [X] Code clearly demonstrates the function.
- [X] All versions maintain the same structure and comments.
- [X] C++ code ran correctly.
- [X] C# top level code ran correctly.
- [X] C# OOP code ran correctly.
- [X] Python code ran correctly.

**Review by huyhuynh2319 — APPROVED** [review]
I reviewed this pull request. Everything looks to be in order, the programs work well 

- [x] C++ code ran correctly.
- [x] C# top level code ran correctly.
- [x] C# OOP code ran correctly.
- [x] Python code ran correctly.

**Review by omckeon — APPROVED** [review]

**Pull request merged**